lms/static/sass/elements-v2/_icons.scss

```diff
@@ -0,0 +1,31 @@
+.xseries-icon {
+  background: url('#{$static-path}/images/programs/xseries-icon.svg') no-repeat;
+}
+
+.micromasters-icon {
+  margin-top: $baseline * 0.05;
+  background: url('#{$static-path}/images/programs/micromasters-icon.svg') no-repeat;
+}
+
+.certificate-icon .certificate-body {
+  @include float(left);
+  @include margin-right($baseline*0.4);
+
+  margin-top: ($baseline/10);
+  width: 23px;
+  height: 20px;
+  padding: 2px;
+  background-color: $white;
+  border-style: solid;
+  border-width: 2px;
```

Comment:
This has also got me thinking about having a reusable ```.green-svg``` or ```.green-icon``` class which could be added to make any icon that same green colour. Thoughts?